db/migrate/20160529163619_rename_booking_id_to_booking_code.rb

```diff
@@ -0,0 +1,5 @@
+class RenameBookingIdToBookingCode < ActiveRecord::Migration
+  def change
+    rename_column :bookings, :booking_id, :booking_code
```

Comment:
**Code Quality - Nitpick**

Change the `booking_id` column to `booking_code` for better readability.